prime-router/src/main/kotlin/fhirengine/engine/FHIRConverter.kt

```diff
@@ -264,6 +301,18 @@ class FHIRConverter(
 
                 // ensure tracking is set
                 actionHistory.trackCreatedReport(nextEvent, report)
+                // TODO: report processing error
+//                azureEventService.trackEvent(
```

Comment:
Why is this commented out? need to update to use reportEventService? It seems like we shouldn't be using AzureEventService anymore? Why are we importing it above?

Reply:
Added a REPORT_PROCESSING_ERROR_EVENT